### PR TITLE
Easier policy setters for ThreadPoolTaskScheduler.

### PR DESCRIPTION
**Pull request by Aelykto**
Added an easier way to set ExecuteExistingDelayedTasksAfterShutdown and ContinueExistingPeriodicTasksAfterShutdown policies. This closes an existing issue: https://github.com/spring-projects/spring-framework/issues/26719"

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@Aelykto Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-framework&pullRequestId=26813)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-framework&pullRequestId=26813) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@Aelykto Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-framework&pullRequestId=26813)!

File: spring-context/src/main/java/org/springframework/scheduling/concurrent/ThreadPoolTaskScheduler.java [hunk 1]
```diff
@@ -64,6 +64,10 @@
 
 	private volatile boolean removeOnCancelPolicy;
 
+	private volatile boolean executeExistingDelayedTasksAfterShutdownPolicy;
```
Comment (cuspymd) [inline spring-context/src/main/java/org/springframework/scheduling/concurrent/ThreadPoolTaskScheduler.java:67]: Default value in `ScheduledThreadPoolExecutor` is `true`. Does it matter if it's different?

Comment (Aelykto) [reply spring-context/src/main/java/org/springframework/scheduling/concurrent/ThreadPoolTaskScheduler.java:67]: I believe you are actually correct and it matters for the getter:
if we don't have a scheduled executor inside our class - we return local mentioned value.

Comment (Aelykto) [reply spring-context/src/main/java/org/springframework/scheduling/concurrent/ThreadPoolTaskScheduler.java]: fyi I added a default true value corresponding to scheduled thread pool executor

**Comment by jhoeller** [comment]
As per my comment on #26719, I ended up with a streamlined arrangement that does not include any policy getter methods.

Thanks for the pull request, in any case!

**Pull request closed without merging**